ashby/amp.yaml

```diff
@@ -0,0 +1,72 @@
+specVersion: 1.0.0
+integrations:
+  - name: readAndWriteAshby
+    displayName: Ashby read and write
+    provider: ashby
+    read:
+      objects:
+        - objectName: application.list
```

Comment:
we should not have verbs in the object name, the support object name should be `application`, not `application.list`. Same for all the other objects here. Can you please revise the connector to use `application` in Read, Write, and ListObjectMetadata methods?